tests/src/runtimeApi/memory/hipMemcpy.cpp

```diff
@@ -292,6 +292,10 @@ void memcpytest2_get_host_memory(size_t& free, size_t& total) {
     MEMORYSTATUSEX status;
     status.dwLength = sizeof(status);
     GlobalMemoryStatusEx(&status);
+    // Windows doesn't allow allocating more than half of system memory to the gpu.
+    // Since the runtime also needs space for its internal allocations,
+    // we should not try to allocate more than 40% of reported system memory,
+    // otherwise we can run into OOM issues.
```

Comment:
In spite of the absence of solid justification for the magic number 40%, it's just a test, thus LGTM.